doc/user/data/sql_funcs.yml

```diff
@@ -356,6 +356,13 @@
     description: 'The `timestamp with time zone` representing when the query was executed.<br/><br/>**NOTE**:
       Users cannot define views with queries containing `now()`.'
 
+  - signature: mz_sleep(d: double precision) -> timestamptz
+    description: |
+      Sleeps during evaluation of the expression for the specified duration in seconds and returns
+      the time at which sleeping finished.  The mz_sleep function is intended to be called
+      interactively, or in transactions during testing. If called in a materialized view, it will be
+      optimized away as aggressively as possible.
```

Comment:
I think I'd actual prefer if we didn't document this! I can't think of a situation where it would be useful to the user and explaining precisely _what_ it blocks while sleeping is tricky.